contrib/examples/apiserver/binding.yaml

```diff
@@ -3,4 +3,9 @@ kind: Binding
 metadata:
   name: test-binding
 spec:
+  instanceRef:
+    name: test-instance
```

Comment:
Good point. I hate to ask the question, but is that something we intend to support? I know this has been discussed before, but I have no recollection of where we've landed.

fwiw, I feel this is an important feature (shared / multi-tenant services, etc.), _however_, I think k8s frowns on this sort of cross-namespace stuff.